pkg/dev/podmandev/pod.go

```diff
@@ -113,7 +113,7 @@ func getVolumeName(volume string, componentName string, appName string) string {
 
 func addHostPorts(containers []corev1.Container, debug bool, usedPorts []int) []api.ForwardedPort {
 	var result []api.ForwardedPort
-	startPort := 40001
+	startPort := 20001
```

Comment:
What do you think of defining both the start and end ports as constants that could be used from the Podman and Kube implementations? To have a single place in case we need to change the range later..

Reply:
I'm not fan of adding coupling between modules if not really coupled. Here, we may want to differentiate podman / cluster ports.

Reply:
Okay - that makes sense.